stf/build.gradle.kts

```diff
@@ -13,12 +13,14 @@ sarosEclipse {
     isAddDependencies = true
     pluginVersionQualifier = versionQualifier
     eclipseVersion = eclipseVersionNr
+    configs = listOf("bundle", "bundleApi")
 }
 
 configurations {
-    val releaseDep by getting {}
+    val bundle by getting {}
+    val bundleApi by getting {}
```

Comment:
Do you need both configurations? `bundle` is unused.

Reply:
True, we have also changed this accordingly.